Makefile

```diff
@@ -38,21 +47,8 @@ ensure-submodule-initialized:
 	else \
 		echo "Submodule found successfully"; \
 	fi
-
-lint-setup-cpp:
-ifeq ("Darwin", "$(shell uname -s)")
-	brew install clang-format
-else
-	sudo apt-get install -y clang-format
-endif
-
-lint-setup-go:
-	GOBIN=`pwd`/$(TOOL_BIN) go install \
-		github.com/edaniels/golinters/cmd/combined \
-		github.com/golangci/golangci-lint/cmd/golangci-lint \
-		github.com/rhysd/actionlint/cmd/actionlint
-
-lint-setup: lint-setup-cpp lint-setup-go
+	grep -q viam-patched viam-cartographer/cartographer/CMakeLists.txt || \
```

Comment:
[note to self] now when submodule is initialized, CMakeList will auto be applied